flang/test/Lower/altret.f90

```diff
@@ -0,0 +1,29 @@
+! RUN: bbc -emit-fir -o - %s | FileCheck %s
+
+function k(n1, n2)
```

Comment:
Do you want a CHECK-LABEL: to anchor your checks against this function?

Reply:
done